### PR TITLE
added skillinator +3

### PR DESCRIPTION
**Pull request by TheDreamSaver**
updated skillinator, jamie's dev shed, & a few other things

Changes proposed in this pull request

- added skillinator
- added indian skill store
- added dev link for echo show
- added java & ruby SDKs

„Be quick, but don't hurry“ - John Wooden
@GetStoryline/awesome-alexa/

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -57,11 +57,13 @@
 * [How to Publish Your Alexa Skill to all the English-speaking Regions?](https://www.youtube.com/watch?v=_Ph6K_-1_mw)
 * [Dabble Lab](https://www.youtube.com/channel/UCfY-LopSxGekh9LruXLjffg/videos) - Weekly videos on building custom skills for Amazon Alexa and related topics.
 * [Building Voice Apps Using Amazon Alexa](https://www.udemy.com/building-voice-apps-using-amazon-alexa/?siteID=Fh5UMknfYAU-NFsHPO_AesnmvLmMUzZEqw&LSNPUBID=Fh5UMknfYAU) - Build smooth voice-activated apps for Echo and other Alexa-aware devices.
+* [Build for Echo Show](https://www.youtube.com/playlist?list=PL4iXGvrBx4ZMPOMDYsbHwjZdaByEL2W2y) - Learn to develope multimodal skills for Echo Show.
```
Comment (MaximAbramchuck) [inline readme.md]: *develop

File: readme.md [hunk 2]
```diff
@@ -120,6 +124,9 @@
 * [VoiceX](https://anilkk.github.io/voicex/) - Prototyping sample dialogs for voice user interfaces.
 * [Sayspring](https://www.sayspring.com) - Create interactive voice prototypes for Amazon Alexa and Google Home.
 * [BotSociety](https://botsociety.io) - Preview & prototype your voice interfaces.
+* [Skillinator](https://skillinator.io/) - Prototype Alexa Skills for Alexa Skills Kit SDK for JavaScript / Node.js (Lambda)
+
```
Comment (MaximAbramchuck) [inline readme.md]: redundant empty line

**Comment by MaximAbramchuck** [comment]
Thanks for the submission, @TheDreamSaver! I've left a couple of comments, I'll merge it as soon as you fix them :)

**Comment by TheDreamSaver** [comment]
updated them.

**Pull request merged**